src/CreepLaw/DiffusionCreep.jl

```diff
@@ -179,10 +183,10 @@ end
 @inline function compute_εII(
     a::DiffusionCreep, TauII::Quantity; T=1K, P=0Pa, f=1NoUnits, d=1e-3m, args...
 )
-    @unpack_units r, p, A, E, V, R = a
+    @unpack_units n, r, p, A, E, V, R = a
     FT, FE = a.FT, a.FE
 
-    ε = A * (TauII * FT) * f^(r) * d^(p) * exp(-(E + P * V) / (R * T)) / FE
+    ε = A * (TauII * FT)^(n) * f^(r) * d^(p) * exp(-(E + P * V) / (R * T)) / FE
```

Comment:
you can use 'fastpow()' here instead of `^` for the powers. same in all the other functions

Reply:
Done.